pkgs/development/libraries/x265/default.nix

```diff
@@ -16,14 +16,14 @@ in
 
 stdenv.mkDerivation rec {
   name = "x265-${version}";
-  version = "2.7";
+  version = "2.9";
 
   src = fetchurl {
     urls = [
       "http://get.videolan.org/x265/x265_${version}.tar.gz"
       "https://github.com/videolan/x265/archive/${version}.tar.gz"
```

Comment:
GitHub archive tarballs are not stable over time. Once it gets regenerated the expected SHA hash won't match.

As an alternative URL we could instead use one of:
`http://ftp.videolan.org/pub/videolan/x265/x265_2.9.tar.gz`
`ftp://ftp.videolan.org/pub/videolan/x265/x265_2.9.tar.gz`
`https://bitbucket.org/multicoreware/x265/downloads/x265_2.9.tar.gz`

Reply:
What about just removing the github link?

Reply:
It would work too yes. But then we lose the insurance of having a fallback server to fetch from.
Not that this is a critical piece of software in the whole `<nixpkgs>` set but some day the fallback could avoid a number of Hydra build failures.